src/components/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const UNDER_CONSTRUCTION = 'Under_Construction';
+const ADD_CATEGORY = 'ADD_CATEGORY';
+
+const categoriesReducer = (state = '', action) => {
```

Comment:
- [ ] Initialize your state to an empty array and set this state to that initial state.

  ```
  const initialState = []
  
  const categoriesReducer = state = initialState, action => {
  // other code
  ```